docs/get_started/index.rst

```diff
@@ -125,9 +125,13 @@ The following steps show how to migrate the Vector Add sample using |tool_name|:
 
       dpct --in-root=. src/vector_add.cu
 
-   The ``--in-root`` option specifies the location of the CUDA file that needs
-   migration. By default, the migrated files are created in a new folder named
-   ``dpct_output``.
+   The ``--in-root`` option specifies the root location of the program sources
+   that should be migrated. Only files and folders located within the
+   ``--in-root`` directory will be considered for migration by the tool. Files
+   located outside the ``--in-root`` directory are considered system files and

```

Comment:
The assumption "Consideered system file" itself maybe not correct, So suggest to remove it.
```suggestion
   located outside the ``--in-root`` directory 

```